src/Underworld.ts

```diff
@@ -3511,6 +3524,8 @@ ${CardUI.cardListToImages(player.stats.longestSpell)}
       case UnitSubType.SUPPORT_CLASS:
         // Support classes (such as priests and summoners) dont attack targets
         return false;
+      case UnitSubType.GORU_BOSS:
+        return false;
```

Comment:
I just saw that goru has the attack badge active in another file because he's always going to do _some_ attack when he has magic so this makes sense now. Please add a comment above this line explaining why it's false.